src/code42cli/main.py

```diff
@@ -53,13 +55,28 @@ def exit_on_interrupt(signal, frame):
 @click.option(
     "--python",
     is_flag=True,
-    help="Print path to the python interpreter env that `code42` is installed in.",
+    help="Print path to the python interpreter env that `code42cli` is installed in.",
+)
+@click.option(
+    "--script-dir",
+    is_flag=True,
+    help="Print the directory the `code42` script was installed in (for adding to your PATH if needed).",
 )
 @sdk_options(hidden=True)
-def cli(state, python):
+def cli(state, python, script_dir):
     if python:
         click.echo(sys.executable)
         sys.exit(0)
+    if script_dir:
+        for root, _dirs, files in os.walk(site.PREFIXES[0]):
```

Comment:
`site-packages` can be global or per-user, so I would expect that you could get rid of the second for loop by just looping through `PREFIXES` instead of only looking at its first element.

Reply:
in all of the installs I've tested, site.PREFIXES had the same value for both items in the list (if there was more than one): 

```
/usr/bin/python -c 'import site; print(site.PREFIXES)'
['/System/Library/Frameworks/Python.framework/Versions/2.7', '/System/Library/Frameworks/Python.framework/Versions/2.7']

python3.9 -c 'import site; print(site.PREFIXES)'
['/Users/tim.abramson/.pyenv/versions/3.9.5', '/Users/tim.abramson/.pyenv/versions/3.9.5']
```
etc.

Reply:
hmm, me too, I must be misunderstanding something about `PREFIXES`. What you have definitely works so I have no problem with getting this out, just thought I was eliminating some code here but I guess not